src/main/resources/application.yaml

```diff
@@ -76,7 +76,7 @@ hapi:
     ### This flag when enabled to true, will avail evaluate measure operations from CR Module.
     ### Flag is false by default, can be passed as command line argument to override.
     cr:
-      enabled: false
+      enabled: true
```

Comment:
**question**: is this an intentional change? I didn't see how it relates to the reported issue.

Reply:
ah good catch, not intentional